sig-contributor-experience/charter.md

```diff
@@ -0,0 +1,59 @@
+# Contributor Experience Special Interest Group Charter
+*checking in for comments and will add TODOs at bottom along with feedback that comes in*
+
+The Contributor Experience Special Interest Group is dedicated to ensuring that Kubernetes contributors have a consistent workflow and great experience within the ecosystem. Developing and sustaining a healthy community of contributors is critical to scaling the project and growing the folks who build it. We champion the needs of these contributors by building programs and processes that address any gaps, learning and development, reward and recognize positive actions, and facilitiate project wide communications. 
+
+## Leads:
+Garrett Rodrigues  
+Paris Pittman  
+Elsie Phillips   
+
+## Processes and Responsibilities
+* Leads run weekly meetings in accordance with [SIG governance procedures](sig-governance.md)
+* Leads run quarterly planning meetings where we decide goals for the next release, to be published in the SIG’s [README](README.md)  
+* SIG Leads plan at least one annual face to face Contributor Experience SIG meeting proposed via mailing list and announced via slack and other channels
+* SIG members run and organize the weekly [Kubernetes Community Meeting](https://goo.gl/zMRLeu)
+* This SIG will act as owners and maintainers of [github.com/kubernetes/community](https://github.com/kubernetes/community)
+* We participate in [lazy consensus](http://en.osswiki.info/concepts/lazy_consensus) for our decisions.
+  * Idea and/or solution is proposed via the kubernetes-wg-contribex@googlegroups.com mailing list. Link(s) to GitHub issue proposal is a nice to have but not required.
+  * Author will give a time period of no less than 3 working days being cognizant of popular observed world holidays to respond with questions, issues, or hold from a lead(s) for more information/research needed.
+  * Author will participate in implementation of the solution
+* We are responsible for community growth and management of Kubernetes and its ecosystem.  We support this growth by creating programs for SIGs, WGs and subprojects to address mentoring, learning and development, contributors, and GitHub workflows.
+  * We will build metrics that capture the project’s velocity and stability.  
+  * This group does not create new SIGs/WGs/subprojects themselves but will mentor new SIGs in the creation and management of their own:
+    * YouTube playlist
+    * Zoom account
+    * Slack channel(s)
+    * Github service groups
+* We ensure communication platforms are a safe environment for discourse and maintain them and in some cases in conjunction with CNCF/LF. Depending on the situation, we will warn first and then report to Steering Committee. Platforms may include Slack, Kubernetes weekly meetings, Zoom, YouTube, etc.
+* Provide guidance and mediation between SIGs/working groups that are having contributor experience issues. For example, consolidation around a certain workflow. 
```

Comment:
Suggest to use "SIGs/WGs"  instead of "SIGs/working groups" to be consistent with overall charter writing style.